pandas/tests/test_categorical.py

```diff
@@ -2951,14 +3002,14 @@ def test_count(self):
         result = s.count()
         self.assertEqual(result, 2)
 
-    def test_sort(self):
+    def test_sort_values(self):
 
         c = Categorical(["a", "b", "b", "a"], ordered=False)
-        cat = Series(c)
+        cat = Series(c.copy())
 
-        # 9816 deprecated
         with tm.assert_produces_warning(FutureWarning):
-            c.order()
+            c.order()  # deprecated in gh-10726
+            c.sort()   # deprecated in gh-12882
```

Comment:
I think you need to do 2 asserts (e.g. say the first one DIDn't assert, but the 2nd one did) (you could do in a loop as well)